Separated buffers (#2)
YuuuZhang/glTF-Asset-Generator: Source/ReadmeTemplates/Buffer_Misc.md

> These models are intended to test using an individual buffer.  

Reviewer: ```suggestion
These models are intended to test edge cases for buffers.  
```

The following table shows the properties that are set for every model.  

> ~~HeaderTable~~

Reviewer: This section is empty. Probably should remove it.

The following table shows the properties that are set for a given model.  

> 

Reviewer: Would recommend removing this text since it doesn't make sense right now.

Reviewer: This line still need to be there or the table won't show up.
